spec/spec_helper.rb

```diff
@@ -15,6 +15,9 @@
 # it.
 #
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
+
+require "rails-controller-testing"
+
```

Comment:
You can probably remove that now, right?

Reply:
True